src/engine/batch_list.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "sm64.h"
+#include "engine/graph_node.h"
+
+struct BatchArray* batch_list_objects_alloc_xlu_decal(struct AllocOnlyPool *pool);
+struct BatchArray* batch_list_objects_alloc_cld(struct AllocOnlyPool *pool);
+
+static inline struct BatchArray* batch_list_objects_alloc(struct AllocOnlyPool *pool, enum RenderLayers layer)
```

Comment:
What's the reasoning behind inlining this, it is just for convenience?

Reply:
It is just for convenience. Compiler will propagate calls to particular cld or xlu_decal call.